docs/getting-started/certificate-pinning.adoc

```diff
@@ -0,0 +1,46 @@
+== Certificate Pinning
+
+Certificate pinning can be enabled in individual SDKs through the `mobile-services.json` file.
```

Comment:
I think we should mention that we are using SPKI pinning somewhere here, and its the only pinning technique supported for internal services.